uint8 lidar_label
float32 range
float32 theta
float32 phi
float32 size 
uint8 num_pixels